### ISSUE TITLE
Object.require is deprecated.

### ISSUE THREAD

**Issue by kerkarpraj** [mikegedelman/atom-force#5]
Use `remote.BrowserWindow` instead of `remote.require("browser-window")`
```
Object.require (C:\Users\Prajyot\AppData\Local\atom\app-1.16.0\resources\app.asar\src\electron-shims.js:54:12)
Object.<anonymous> (C:\Users\Prajyot\.atom\packages\atom-force\lib\atom-force.js:55:41)
Module._compile (C:\Users\Prajyot\AppData\Local\atom\app-1.16.0\resources\app.asar\src\native-compile-cache.js:109:30)
Object..js (C:\Users\Prajyot\AppData\Local\atom\app-1.16.0\resources\app.asar\src\compile-cache.js:216:21)
Module.load (module.js:473:32)
tryModuleLoad (module.js:432:12)
```

**Comment by Priya-Sundaram** [comment]
Facing the same issue..

**Comment by rmx90210** [comment]
Same, it won't open a browser window so I can't activate the plugin :/

**Comment by mikegedelman** [comment]
I might have some time to take a look at it this week, but feel free to make a PR too!

**Comment by rmx90210** [comment]
Taking a stab at it now :)

**Comment by mikegedelman** [comment]
A patch has been published to fix this, v0.3.6.
